docs/CONTRIBUTING.md

```diff
@@ -12,22 +12,52 @@ request.
 
 It is important to note that there are **two types of PRs** for this repository:
 
-1. **New Userstyles:** Please see [userstyle-creation.md](userstyle-creation.md).
-2. **Other improvements:** These should follow the recommendations below, to help
-   improve the chances of a merged PR.
-
-## General Recommendations
-
-- Create a
-  [topic branch](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows#_topic_branch)
-  on your fork for your specific PR.
-- Catppuccin uses the [Conventional Commits
-  ](https://www.conventionalcommits.org/en/v1.0.0/) standard for creating explicit and meaningful commit messages. This repository requires pull request _titles_ to be in the conventional commit format, however we do not require it for individual commits within a pull request.
-- Update the version in the `==UserStyle==` header of the `catppuccin.user.css` file. This is to enable version control of the style.
+1. **New Userstyles:** Please see [userstyle-creation.md](./userstyle-creation.md).
+2. **Other improvements:** These should follow the recommendations below, to help improve the chances of a merged PR.
+
+## Recommendations
+
+### Root Variables
+
+While writing a userstyle, you may have come across [custom properties / CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*), typically wrapped in a `:root` selector. We refer to these variables as "root variables", and they are sort of global variables used all across a website. For Catppuccin userstyles, we prefer that these variables are themed rather than individual elements. As the website likely uses these variables themselves, it saves yourself a lot of work in theming and maintaining the userstyle.
```

Comment:
```suggestion
While writing a userstyle, you may have come across [custom properties / CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*), typically wrapped in a `:root` selector. We refer to these variables as "root variables", and they can be though of as global variables used all across a website. For Catppuccin userstyles, we prefer that these variables are themed (if they exist) rather than individual elements. As the website hopefully uses these variables themselves, it saves yourself a lot of work in theming and maintaining the userstyle.
```